rpcs/eth/EthRpc.js

```diff
@@ -119,6 +119,22 @@ class EthRPC {
     });
   }
 
+  async unlockAndSendToAddressMany({ payToArray, passphrase }) {
+    let phrase = passphrase;
+    if (passphrase === undefined) {
+      try {
+        phrase = await promptly.password('> ');
+      } catch (e) {
+        return Promise.reject(e);
+      }
+    }
+    const promises = payToArray.map(a => {
+      const [address, amount] = a;
+      return this.sendToAddress({ address, amount, phrase });
+    });
+    return Promise.all(promises);
```

Comment:
I don't think that Promise.all gives us the behavior that we want... if one of the transactions fails then it would reject without giving us information about any that succeeded